doc/cortx-aws-k8s-installation.md

```diff
@@ -224,7 +224,7 @@ At this stage the Kubernetes cluster should be fully operational
 
 ### 3.1 Clone Cortx-K8s framework
 ```
-git clone -b stable https://github.com/Seagate/cortx-k8s.git
+git clone -b integration https://github.com/Seagate/cortx-k8s.git
```

Comment:
You could remove the `-b integration` since this is the default branch (as of today), however, we shouldn't recommend its use anyways since it's basically a development branch. Instead, I'd either update it to point to a release tag or change it to download one of the release archive files.

Reply:
I switched it to use the v0.0.22 tag